### PR TITLE
installer: add grub.cfg for EFI boot from ISO

### PR DESCRIPTION
**Pull request by dustymabe**
- installer: now using an images and isolinux sub dir
- installer: add grub.cfg for EFI boot from ISO

### CODE REVIEW

File: installer/EFI/fedora/grub.cfg [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+set gfxpayload=keep
+insmod gzio
+insmod part_gpt
+insmod ext2
+
+set timeout=60
+### END /etc/grub.d/00_header ###
+
+### BEGIN /etc/grub.d/10_linux ###
+menuentry 'Install Fedora CoreOS' --class fedora --class gnu-linux --class gnu --class os {
```
Comment (cgwalters) [inline installer/EFI/fedora/grub.cfg]: Wondering whether we should instead ship this with cosa and just substitute the name from the manifest.

Comment (cgwalters) [reply installer/EFI/fedora/grub.cfg]: (This applies to the other stuff already here in `installer/` of course...not a blocking issue but let's at least consider it?)

Comment (dustymabe) [reply installer/EFI/fedora/grub.cfg]: I personally don't think so, but I think it's reasonable to have that conversation in a ticket (or meeting) dedicated for that discussion to get input from others.

Comment (cgwalters) [reply installer/EFI/fedora/grub.cfg]: OK we can change it later.

**Pull request merged**